lib/go-atscfg/strategiesdotconfig.go

```diff
@@ -256,6 +275,22 @@ func getStrategyHostsSection(pa *ParentAbstraction) string {
 			txt += getStrategyHostsSectionHost(svc, parent)
 		}
 	}
+	if len(pa.Peers) != 0 {
+		count := 1
+		for _, peer := range pa.Peers {
```

Comment:
Nitpick: could use the loop index to avoid a separate variable here, ` for i, peer := range pa.Peers { txt += getStrategyHostsSectionPeer(peer, i+1)`